pkg/cvo/cvo_test.go

```diff
@@ -13,7 +13,6 @@ import (
 	"github.com/davecgh/go-spew/spew"
 	"github.com/golang/glog"
 	"github.com/google/uuid"
-
```

Comment:
nit: why this drop?

Reply:
it's vscode go tools being stupid, sorry